pkgs/shells/bash/ble.sh/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib, stdenvNoCC, fetchgit, git, curl, jq, nix-prefetch, runtimeShell }:
+
+stdenvNoCC.mkDerivation {
+  name = "ble.sh";
+  version = "0.3.3";
+
+  src = fetchgit {
+    url = "https://github.com/akinomyoga/ble.sh.git";
+    rev = "refs/tags/v0.3.3";
+    hash = "sha256-Gfo2S1t5Kdy+8TEDS4M5yhyRShvzQIljdE0MQK1CL+4=";
+    fetchSubmodules = true;
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [ git ];
+  buildInputs = [ curl jq nix-prefetch ];
+
+  doCheck = true;
+
+  installFlags = [ "INSDIR=$(out)/share" ];
+  postInstall = ''
+    mkdir -p "$out/bin"
+    cat <<SCRIPT >"$out/bin/ble.sh-share"
+    #!${runtimeShell}
+    # Run this script to find the ble.sh shared folder
+    # where all the shell scripts are living.
+    echo "$out/share/ble.sh"
+    SCRIPT
```

Comment:
I don't think we need this at all. We can just source the files inside of ``programs.bash.interactiveShellInit`` where we can refer to $pkgs.ble-sh.

Reply:
Not all the users are using configuration.nix, and this way is described on the nixpkgs docs. That is why I assumed it is the current standard way of packaging programs like ble.sh.

Reply:
Where is that described in the docs?

If you are not on NixOS you install this with either home-manager or nix-env and would source it from the symlink in your home directory.

Reply:
Here. https://nixos.org/manual/nixpkgs/stable/#sec-shell-helpers